### PR TITLE
Desktop version

### PR DESCRIPTION
**Pull request by AlexHamn**
The whole website now responds to a larger display.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -68,87 +75,94 @@ <h1>Alex Hamn</h1>
...
+            <button>See Project</button>
+          </div>
         </article>
       </section>
       <section class="container contact">
         <h2>CONTACT US</h2>
+        <h2>Let's chat!</h2>
         <p> 
           If you have an application you are interested in developing, a feature
           that you need built or a project that needs coding. I’d love to help
```
Comment (Shaher-11) [inline index.html:168]: Reduce the width of the form below
### Original design
right

### Your Project
wrong

File: index.html [hunk 2]
```diff
@@ -68,87 +75,94 @@ <h1>Alex Hamn</h1>
...
+              <li>bootstrap</li>
+              <li>Ruby</li>
+            </ul>
+            <button>See Project</button>
+          </div>
         </article>
       </section>
       <section class="container contact">
         <h2>CONTACT US</h2>
+        <h2>Let's chat!</h2>
```
Comment (Shaher-11) [inline index.html:165]: Add some space between the paragraph and the form below

File: index.html (hunk 1 above)
Comment (Shaher-11) [reply index.html:168]: Center the button as shown below
### Original design 
right

### Your Project
wrong

File: index.html [hunk 3]
```diff
@@ -68,87 +75,94 @@ <h1>Alex Hamn</h1>
...
-          </ul>
-          <button>See Project</button>
+          <img src="https://picsum.photos/684/448?grayscale" alt="placeholder"/>
+          <div class="container project-text">
+            <h3>Multi-Post Stories</h3>
+            <p>
+              A daily selection of privately personalized reads; no accounts or
+              sign-ups required. has been the industry's standard dummy text ever
+              since the 1500s, when an unknown printer took a standard dummy text.
+            </p>
```
Comment (Shaher-11) [inline index.html:98]: Increase the width for each one of the projects paragraphs

**Review by Shaher-11 — CHANGES_REQUESTED** [review]
#### Hello @AlexHamn, 

### Great job so far!
You have been doing an incredible  job on the project so far:clap::clap:
The project is very well structured:smiley::smiley:
However, we still need to make a few changes to get the project approved:muscle:

 ## Required Changes ♻️

- [ ] Check the comments below:point_down:


_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Happy Coding!👏👏👏

File: index.html (hunk 3 above)
Comment (AlexHamn) [reply index.html:98]: It seems like all the project paragraphs are of an adequate width; they're even a few pixels larger than the original template. 😄 
So, could you please explain to me what it is exactly what you mean by this comment? 🙏🏼

Comment (Shaher-11) [reply index.html:98]: Each one of the projects has a paragraph on the side you need to increase the width to reduce the empty space a little bit

Comment (AlexHamn) [reply index.html:98]: Got it, ty 🙏🏼 😁

**Review by Shaher-11 — APPROVED** [review]
#### Hello @AlexHamn,

# Congratulations! 🎉 
#### Your project is approved!  There is nothing else to say other than... it's time to merge it :muscle: :muscle::muscle:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Cheers and Happy coding!👏👏👏

**Pull request merged**